api/appendix_e.asciidoc

```diff
@@ -367,8 +367,8 @@ device:
     collective functions, such as broadcasts, scans, and reductions.
   * {CL_DEVICE_GENERIC_ADDRESS_SPACE_SUPPORT} to
     determine whether a device supports the generic address space.
-  * {CL_DEVICE_DEVICE_ENQUEUE_SUPPORT} to determine whether a device
-    supports device queues and device-side enqueue.
```

Comment:
Nitpick: dropped the `.` at end of sentence, and in `api/opencl_platform_layer.asciidoc` too